spec/sections/usage.adoc

```diff
@@ -17,10 +17,11 @@ MUST follow Bash expansion conventions.
 The following minimum variables MUST be provided:
 
 * JOB_OUTPUT_DIR: Absolute path where all output products MUST be placed by job for Seed executor capture.
-* ALLOCATED_CPUS: Value of `cpus` member.
-* ALLOCATED_MEM: Value of `mem` member.
-* ALLOCATED_SHARED_MEM: Value of `sharedMem` member.
-* ALLOCATED_STORAGE: Value of `storage` member.
+* All `name` member values of `resources.scalar` elements MUST map to environment variables. This SHALL be done by
+normalizing all `name` values to uppercase, stripping all non-alphabetic characters and prefixing with `ALLOCATED_`.
+For a `my-demo-resource5` with a value of `5.0`, an environment variable `ALLOCATED_MYDEMORESOURCE` SHALL be set to
+`5.0` at run time. For a resource with an `inputMultiplier` value the environment variable must include the final
+computed resource allocation as defined in the `scalar` element under <<resources-section,Resources object>>.
```

Comment:
We should clarify between total allocated disk, disk allocated for input, and disk allocated for output. For example, the formula (const + mult * input size) calculates the needed disk for OUTPUT; you must still add the input size again for total disk required by the algorithm. I would think we would want the env var to reflect the total disk space (input + output) needed. We could have multiple env vars for disk too.